### ISSUE TITLE
jailing not tping people to jail

### ISSUE THREAD

**Issue by higuy7634dev** [Sc0ttM/SEM_InteractionMenu#13]
jailing not tping people to jail and they can leave aat anytime

**Comment by Sc0ttM** [comment]
There have been a few reports of this so I looked over the code for both the Jail and Hospital feature and found an issue.

in the `client.lua` at **line 335** there is the following
```
if IsPedInAnyVehicle(Player, false) then
     ClearPedTasksImmediately(Player)
end
```
you need to replace the "Player" with "Ped" there is the same section of code for the hospital feature at **line 506**

**Comment by Sc0ttM** [comment]
This issue should be fixed in the latest update **v1.6**
